float64 aile_av
float64 aile_ar
float64 epaule_av
float64 epaule_ar
float64 coude_av
float64 coude_ar